front/cypress/e2e/admin/pages_and_menu/update_homepage_hero_banner_content.cy.ts

```diff
@@ -77,6 +77,30 @@ describe('Admin: update Hero Banner content', () => {
     );
   });
 
+  it('uploads and crops banner image', () => {
+    cy.intercept('PATCH', '**/home_page').as('saveHomePage');
+    cy.setAdminLoginCookie();
+    cy.visit('admin/pages-menu/homepage/homepage-banner/');
+    cy.acceptCookies();
+
+    cy.get('[data-cy="e2e-fixed-ratio-layout-option"]').click();
+    cy.get('[data-cy="e2e-remove-image-button"]').click();
+    cy.get('[data-cy="e2e-homepage-banner-image-dropzone"] input').attachFile(
+      'testimage.png'
+    );
+
+    // I see the change looking at the Cypress Studio, but I'm not sure how to test it.
```

Comment:
Test which change?

Movement/functioning of the image cropper? Maybe that shouldn't be in the E2E test but rather in a unit/integration test?

Reply:
> Movement/functioning of the image cropper?

Yes.

> Maybe that shouldn't be in the E2E test but rather in a unit/integration test?

Then what should be in this e2e test? Should we just remove it?

Reply:
> Then what should be in this e2e test? Should we just remove it?

Testing that the image cropper works. So uploading and saving the image does that.

> Should we just remove it?

I believe so, since we're not verifying the result of this action.

Not sure what more to test here. Maybe @IvaKop has a better idea here?

Reply:
Sure, let's add more unit tests. I just want to focus specifically on e2e tests in this task and this PR.

> drag, file uploads, etc.

I tested dragging, but I removed it because I couldn't figure out how to check that it really works
https://github.com/CitizenLabDotCo/citizenlab/pull/3544/commits/87fd001e41ca1aee0df18aca9c8f4e2e21156fc7#diff-af9b2e9377728584552837046f7df9a2a244d950487fd014490b79f3cd359d1eR93
Maybe I could crop the image with some independent tool and then compare it to what is displayed in the browser?

> navigate to where I can see it - do I see what I expect?

Similar to the previous question. How could I check that the image is really cropped?

Reply:
Good question! Not sure how to test the drag + crop actually in a reliable way. But also, perhaps it's not that important to do so because this functionality comes directly from the library we use and we should normally test our own logic, not the external code.

> Similar to the previous question. How could I check that the image is really cropped?
I think checking that the ratio and display are correct should be enough. You can make sure you upload an image with the "wrong" ratio so if it's correct in the display - it has been cropped.